variables.tf

```diff
@@ -1,7 +1,22 @@
-variable "admin_account_ids" {
-  type        = list(string)
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+
+variable "admin_account_id" {
+  type        = string
   description = "A list of AWS account IDs to associate with AWS Firewall Manager as the AWS Firewall Manager administrator account. This can be an AWS Organizations master account or a member account. Defaults to the current account."
-  default     = []
+}
+
+variable "is_destroy" {
```

Comment:
It seems like this is the same as `enabled=false`

Reply:
This var is actually to support `destroy`ing - since you need a specific assumed role to destroy thats different from create. https://github.com/hashicorp/terraform-provider-aws/issues/19928